Dockerfile

```diff
@@ -1,12 +1,12 @@
 ############# builder
-FROM golang:1.16.3 AS builder
+FROM golang:1.17.5 AS builder
 
 WORKDIR /go/src/github.com/gardener/gardener-extension-provider-aws
 COPY . .
 RUN make install
 
 ############# base
-FROM alpine:3.13.2 AS base
+FROM alpine:3.15.0 AS base
```

Comment:
Months ago (July this year) with @vpnachev we experienced issues with `alpine@3.14` where 3.14 was working only with latest versions of runc (`>=v1.0.0-rc93`). The runc version in the concourse workers was older one. And the runc version in garden-linux@184.0 was an older one. Are these issues now resolved and safe to update to `alpine@3.15.0`?

/cc @AndreasBurger

Reply:
Can you share more information what these problems were (is there an issue or something about this)?

Reply:
> Can you share more information what these problems were (is there an issue or something about this)?

I forwarded you the corresponding Slack conversation back then. Unfortunately, I don't recall all of the details. 
If we have components that are already using `alpine@3.15` for some time without any issues (wrt to concourse and garden-linux), then this change is fine for me. 👍

Reply:
external-dns-management was recently updated to `alpine@3.15` (ref https://github.com/gardener/external-dns-management/pull/233). Looks like no issues occurred during its release and no issues so far on landscapes. Then I guess it is fine to proceed with `alpine@3.15`.

Reply:
Hm, the PR was only merged yesterday, so I guess we won't have much experience yet. I tend towards reverting the alpine changes for now asp art of this PR. WDYT?

Reply:
Okay, we can delay the alpine version update and do it when we have more experience with it (we will monitor external-dns-management).

Reply:
👍🏻 Thanks :)